File: game-server/src/main/java/org/forfun/mmorpg/game/scene/model/AoiObserver.java
package org.forfun.mmorpg.game.scene.model;

import java.util.HashMap;
import java.util.Map;

import org.forfun.mmorpg.game.scene.actor.SceneActor;

/**
 * aoi分屏算法视野观察者
 * 
 *
 */
public class AoiObserver {

	/** 当前可见视野元素 */
	private Map<Long, SceneActor> actors = new HashMap<>();

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/scene/model/AoiObserver.kt
package org.forfun.mmorpg.game.scene.model

import org.forfun.mmorpg.game.scene.actor.SceneActor

/**
 * aoi分屏算法视野观察者
 */
class AoiObserver {

    /** 当前可见视野元素 */
    val actors = HashMap<Long, SceneActor>()
}
